p4-16/spec/P4-16-spec.mdk

```diff
@@ -4210,11 +4210,11 @@ Given a header stack value `hs` of size `n`, the following
 expressions are legal:
 
 - `hs[index]`: produces a reference to the header at the
-  specified position within the stack; if `hs` is an l-value, the
-  result is also an l-value. The header may be invalid. Some implementations
+  specified position within the stack; if `hs` is an l-value,
+  the result is also an l-value. The header may be invalid. Some implementations
   may impose the constraint that the index expression evaluates to a
-  compile-time known value. A P4 compiler must give an error if an
-  index value that is a compile-time constant is out of range.
+  value that is known at compile time. A P4 compiler must give an error
```

Comment:
Actually this is a term that we tried to use consistently through the document: "compile-time known value".

Reply:
Yes, but it`s poor English.

I`m guessing nobody other than myself would use “{compilation time}-known value” or “compileTime-known value”, so I tried to come up with a semantically-equivalent phrase that is good English.